src/data/complete.queryMap.json

```diff
@@ -1519,8 +1519,8 @@
   "3016be253f8faa7f4a0720968b41dd13": "query FeaturedArtistsTestsQuery {\n  marketingCollection(slug: \"emerging-photographers\") {\n    ...FeaturedArtists_collection\n    id\n  }\n}\n\nfragment FeaturedArtists_collection on MarketingCollection {\n  artworksConnection(aggregations: [MERCHANDISABLE_ARTISTS], size: 9, sort: \"-decayed_merch\") {\n    merchandisableArtists {\n      slug\n      internalID\n      name\n      image {\n        resized(width: 500) {\n          url\n        }\n      }\n      birthday\n      nationality\n      isFollowed\n      id\n    }\n    id\n  }\n  query {\n    artistIDs\n    id\n  }\n  featuredArtistExclusionIds\n}\n",
-  "0b90b9c3923995ca0fd4fa8d509f7be9": "query AutosuggestResultsQuery(\n  $query: String!\n  $count: Int!\n) {\n  ...AutosuggestResults_results_1bcUq5\n}\n\nfragment AutosuggestResults_results_1bcUq5 on Query {\n  results: searchConnection(query: $query, mode: AUTOSUGGEST, first: $count, entities: [ARTIST, ARTWORK, FAIR, GENE, SALE]) {\n    edges {\n      node {\n        imageUrl\n        href\n        displayLabel\n        ... on SearchableItem {\n          displayType\n          slug\n        }\n        ... on Node {\n          id\n        }\n        __typename\n      }\n      cursor\n    }\n    pageInfo {\n      endCursor\n      hasNextPage\n    }\n  }\n}\n",
```

Comment:
@ashfurrow this looks like a bot PR that tries to undo our artisanal changes, yes?

I'd say maybe we should just close it but there's another non-search query impacted down below 🤔

Reply:
Argh, gotcha. That's because this isn't merged yet so the query map is regenerating as-is: https://github.com/artsy/emission/pull/2075

We'll need to undo this change (ie: re-do the earlier change) once that PR gets merged. Does that make sense?

Reply:
Yes, thanks for confirming 👍